docs/instance-plugin.md

```diff
@@ -403,3 +404,21 @@ gradlew instanceGroovyEval -i -Pinstance.groovyEval.script=iteration-17/* -Pinst
 
 By default, any failed script evaluation will cause that task will also fail.
 To avoid that (ignore failures), add parameter `-Pinstance.groovyEval.faulty=false`.
+
+
+## Task `instanceWorkflow`
```

Comment:
I am thinking that maybe it should be also possible to run workflow once at concrete path.
We should implement it as well